aviatrix/utils.go

```diff
@@ -196,3 +200,97 @@ func DiffSuppressFuncGatewayVpcId(k, old, new string, d *schema.ResourceData) bo
 
 	return false
 }
+
+func reverseArray(arr []string) []string {
+	for i, j := 0, len(arr)-1; i < j; i, j = i+1, j-1 {
+		arr[i], arr[j] = arr[j], arr[i]
+	}
+	return arr
+}
+
+// sort the firewall_image_version list
+func sortVersion(versionList []string, i, j int) bool {
+	if checkFirstCharacter(versionList[i]) == "R" {
```

Comment:
We probably should check the name of the firewall provider instead of jus the first character. That way we won't run into issues if a new format is added.

Reply:
i check the name of firewall_image, but find this format version: R81.10-335.986 firewall_image name is too same as Semantic Version firewall_image name, it is hard to find a way to distinguish them.

Semantic Version：
Check Point CloudGuard IaaS Single Gateway R80.30 - Bring Your Own License
Check Point CloudGuard IaaS Single Gateway R80.30 - Pay As You Go (NGTP)
Check Point CloudGuard IaaS Single Gateway R80.30 - Pay As You Go (NGTX)
Check Point CloudGuard IaaS Single Gateway R80.40 - Bring Your Own License
Check Point CloudGuard IaaS Single Gateway R80.40 - Pay As You Go (NGTP)
Check Point CloudGuard IaaS Single Gateway R80.40 - Pay As You Go (NGTX)
Check Point CloudGuard IaaS Standalone (gateway + management) R80.40 - Bring Your Own License

R81.10-335.986：
Check Point CloudGuard IaaS Next-Gen Firewall with Threat Prevention
Check Point CloudGuard IaaS All-In-One
Check Point CloudGuard IaaS Firewall & Threat Prevention (Gateway only)
Check Point CloudGuard IaaS Firewall & Threat Prevention (Gateway only) (BYOL)
Check Point CloudGuard IaaS Firewall & Threat Prevention (Standalone)
Check Point CloudGuard IaaS Firewall & Threat Prevention (Standalone) (BYOL)

Reply:
have fixed base on the image name to choose the sort function